### DISCUSSION TITLE
How to embed CMS in Microsoft Teams

### DISCUSSION THREAD

**Discussion by saulobrito01** [OrchardCMS/OrchardCore#12526] [Show and tell]
How to embed CMS in Microsoft Teams

Would it be possible to embed the CMS within Microsoft Teams?

For example, when logging in through Teams and accessing a tab with the application, you will automatically log in to the application;

There is a possibility? At least the possibility to embed the CMS

**Comment by hishamco** [comment]
If Teams support embedded HTML this will be possible. I think you should move your question into Microsoft Teams repo if there's while it's not related to OC

**Comment by ns8482e** [comment]
@saulobrito01 you can create teams app and connect your app to OC via api/graphql using open id
